.semaphore/semaphore.yml

```diff
@@ -9,9 +9,6 @@ global_job_config:
       value: v2.0.2
   prologue:
     commands:
-      - export HOME=$WORKSPACE
-      - mkdir $WORKSPACE/confluent-kafka-python
-      - cd $WORKSPACE/confluent-kafka-python
```

Comment:
These 3 commands can be removed, as semaphore provided command `checkout` does all of this.